code/Makefile

```diff
@@ -157,6 +159,10 @@ $(filter %$(GRAPH_P_SUFFIX), $(GRAPH_PACKAGES)): %$(GRAPH_P_SUFFIX): check_stack
 graphs: $(GRAPH_PACKAGES)
 ###
 
+analysis:
```

Comment:
This target should have the `check_stack` dependency.

Reply:
so `analysis` should instead be `analysis: check_stack`?

Reply:
Yup.